Articles/ShortArticles/MythOfDeveloperProductivity.md

```diff
@@ -0,0 +1,40 @@
+# Myth of Developer Productivity
+
+<!--deck text start-->
+The ability to directly effectively measure individual software developer productivity has eluded software developers for decades.  Why is this so hard and what are the alternatives? 
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe), [Mark C. Miller](https://github.com/markcmiller86)
+
+#### Publication date: December, ??? 2022
+
+Here are the references I want to include: [myth-dev-prod], [cant-meas-prod], [kanban-and-scrum], [impl-lean-dev] (NOTE: We need a CC article for [impl-lean-dev].)
+
+Key points to make?
+
+* Why we care to measure productivity? => Software estimation, individual developer performance evaluation?
```

Comment:
Always be improving processes is another reason to care. If we don't know how much something costs, we can't begin to identify where we could reduce cost (e.g. improve productivity).

What about how we (in the HPC/CSE space anyways) don't really seem to be charged with *just writing code* or *just designing* or *just testing* or just *product development* or *just support*. All of us wear all of these hats and frequently context switch them multiple times in any given day making the problem of measuring cost of any one that much harder.

Reply:
Again, we can measure things that we know are associated with waste and work to eliminate those things.  And context switching and relearning is a major form of waste.